doc/admin/pool_operations.md

```diff
@@ -40,7 +42,7 @@ $ dmg pool create --help
       -P, --properties= Pool properties to be set
       -a, --acl-file=  Access Control List file path for DAOS pool
       -z, --size=      Total size of DAOS pool (auto)
-      -t, --scm-ratio= Percentage of SCM:NVMe for pool storage (auto) (default: 6)
+      -t, --tier-ratio= Distribution of pool storage allocation over storage tiers (auto) (default: 6)
```

Comment:
alignment of columns has changed for this line and is now off, not sure whether that matters